org.civicrm.reqangular/test/decorators/uib-tooltip.decorator.spec.js

```diff
@@ -0,0 +1,230 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angularMocks',
+  'common/angular',
+  'common/decorators/uib-tooltip.decorator',
+  'common/lodash',
+  'common/angularBootstrap'
+], function (angularMocks, angular, $uibTooltipDecorator, _) {
+  'use strict';
+
+  describe('$uiTooltip.clickable', function () {
+    var $compile, $document, $rootScope, $triggerElement, $timeout, $tooltipElement, $provide;
+
+    beforeEach(function () {
+      module('ui.bootstrap');
+      module(function (_$provide_, _$uibTooltipProvider_) {
+        $provide = _$provide_;
+
+        _$uibTooltipProvider_.setTriggers({ 'customShow': 'customHide' });
+      });
+      inject(function () {
+        $provide.decorator('$uibTooltip', $uibTooltipDecorator);
+      });
+    });
+    beforeEach(inject(function (_$compile_, _$document_, _$rootScope_, _$timeout_) {
+      $compile = _$compile_;
+      $document = _$document_;
+      $rootScope = _$rootScope_;
+      $timeout = _$timeout_;
+    }));
+
+    describe('when a document with a tooltip with the "uib-tooltip-clickable" attribute is opened', function () {
+      beforeEach(function () {
+        cleanDocument();
+
+        $triggerElement = angular.element('<div uib-tooltip-template="\'tooltip\'" uib-tooltip-clickable="true" tooltip-append-to-body="true"><script id="tooltip" type="text/ng-template"><div class="tooltip-clickable-template">Content</div></script></div>');
```

Comment:
if we could split this element definition into multiple lines it would be awesome

Reply:
Done.